app/models/miq_widget.rb

```diff
@@ -492,6 +492,10 @@ def self.sync_from_hash(attrs)
     widget
   end
 
+  def filter_for_schedule
+    "\"=\" => {\"field\" => \"MiqWidget-id\", \"value\" => #{id}}"
```

Comment:
Might be clearer to just use `%Q[]` instead of double quotes.

Reply:
👍  this one ``` %Q["=" => {"field" => "MiqWidget-id", "value" => #{id}}]``` looks much cleaner

Reply:
I do not want to fix style warning, as of now looks more clear